recorder/src/main/cpp/circular_queue.cc

```diff
@@ -48,7 +50,7 @@ bool CircularQueue::pop() {
         usleep(1);
     }
 
-    listener_.record(buffer[current_output].trace, buffer[current_output].info);
+    listener_.record(buffer[current_output].trace, buffer[current_output].info, buffer[current_output].ctx_len, &buffer[current_output].ctx);
     
     // 0 out all frames so the next write is clean
     JVMPI_CallFrame *fb = frame_buffer_[current_output];
```

Comment:
Didn't understand this one. We are not zero-initializing anything here. We are just passing current-output-idx buffer element into the listener.